### PR TITLE
Userland+LibFSAC: Filter out unsupported file types in open dialogs in more apps

### PR DESCRIPTION
**Pull request by krkk**
<table>
<tr>
<td>
<td>
<td>

<tr>
<td>
<td>
<td>

<tr>
<td>
<td>
<td>

<tr>
<td>

</table>

### CODE REVIEW

File: Userland/Libraries/LibFileSystemAccessClient/Client.h [hunk 1]
```diff
@@ -48,6 +48,13 @@ class File {
 
 using Result = ErrorOr<File>;
 
+struct OpenFileOptions {
+    DeprecatedString window_title = {};
```
Comment (LucasChollet) [inline Userland/Libraries/LibFileSystemAccessClient/Client.h:52]: I think that this one can be a `StringView`

**Review by LucasChollet — COMMENTED** [review]
Small nit, otherwise LGTM!

**Comment by AtkinsSJ** [comment]
Very nice! I'm noticing in some of those screenshots though, some file types are "Foo Files" and some are "Foo File". The consistency hobgoblin in my head would like those to be the same please. :sweat_smile:

**Review by AtkinsSJ — APPROVED** [review]
👍

**Pull request merged**